src/time/BrokenDate.hpp

```diff
@@ -116,6 +121,10 @@ struct BrokenDate {
 
   void DecrementDay();
 
+  StaticString<20> c_str();
```

Comment:
This is called `c_str` but it doesn't return a C string. Confusing.

Reply:
Removed